frontend/src/pages/api/nodes.js

```diff
@@ -2,15 +2,23 @@ import { ExplorerApi } from "../../libraries/explorer-wamp";
 
 export default async function (req, res) {
   try {
-    await new ExplorerApi(req).call("node-telemetry", [
+    let ip_address =
+      req.headers["x-forwarded-for"] ||
+      req.connection.remoteAddress ||
+      req.socket.remoteAddress ||
+      req.connection.socket.remoteAddress;
```

Comment:
Are those different?

Reply:
I search and find this for some browser may cause problem if we do not have all these. this will make sure all browser can give correct location.